app/services/album.js

```diff
@@ -1,6 +1,14 @@
 const axios = require('axios');
+const logger = require('../logger');
+const errors = require('../errors');
 
 exports.albums = () =>
-  axios.get('https://jsonplaceholder.typicode.com/albums').then(response => {
-    console.log(response);
-  });
+  axios
+    .get('https://jsonplaceholder.typicode.com/albums')
```

Comment:
I forgot to highlight this in the previous review, but `https://jsonplaceholder.typicode.com` is the same for the entire service and can be stored in a `const` for reuse.